docs/sphinx/about/whats-new.txt

```diff
@@ -1,6 +1,25 @@
 Version history
 ===============
 
+5.3.0-m2 (2016 November 23)
```

Comment:
Either way. The advantage of combining early is that we could cut some redundancy e.g. I wonder whether the "clean up..."  should be mentioned at all in a combined history.